docs/Installation.md

```diff
@@ -11,7 +11,12 @@ micro-controllers,
 [ARM based micro-controllers](Features.md#step-benchmarks), and
 [Beaglebone PRU](beaglebone.md) based printers.
 
-## Prepping an OS image
+## Use pre-installed ready to use image
+You can download a pre-installed image with OctoPi and Klipper with the correct default settings.
+
+[OctoPi-Klipper can be downloaded here](https://github.com/guysoft/OctoPi-Klipper).
```

Comment:
Then maybe add [Maisail](https://docs.mainsail.xyz/) and [fluidd](https://docs.fluidd.xyz/)?

Reply:
Possible,  for now I wanted to fix the out-of-date octoprint instructions. Which is what this pr does


Once its accepted adding others makes sense too.